### DISCUSSION TITLE
Change text color on the geomap panel

### DISCUSSION THREAD

**Discussion by ethaniel** [grafana/grafana#63949] [Dashboards, Panels & Visualizations]
It would be awesome to be able to change the text color of the markers on the geomap panel.
Right now, I color the markers via thresholds, however it would be much nicer if the text itself would be white while the marker would still pick the color up from the threshold value.

Right now the text shares the same color as the marker:
<img width="448" alt="image" src="https://user-images.githubusercontent.com/941973/222185965-53f2419a-2024-4c98-8c35-94fcfad9e20b.png">

Thank you!

**Comment by ethaniel** [comment]
Solved for now by creating an additional cloned layer with only text:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/941973/222189899-1b738d20-96ae-435f-8437-70c1709f9b8b.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82788. Feel free to continue the discussion around this there. Thank you!
